lib/utils/replacements-utils.ts

```diff
@@ -19,15 +19,8 @@ type BaseReferenceElement<T> = {
 
 export function parseReplacementsForString(
     text: string,
-): BaseReplacementElement<{
-    /* empty object */
-}>[] {
-    return baseParseReplacements<
-        {
-            /* empty object */
-        },
-        { value: string }
-    >(
+): BaseReplacementElement<object>[] {
+    return baseParseReplacements<object, { value: string }>(
```

Comment:
These were caught by the new [`@typescript-eslint/no-empty-object-type`](https://v8--typescript-eslint.netlify.app/rules/no-empty-object-type). `object` is a more correct representation of what these are looking for: empty objects (not allowing primitives).